week-1/2-mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return `${word.trim()}`;
```

Comment:
Nice fix! You could just return `word.trim()` here since you don't need to add anything to the string.